staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -775,6 +781,9 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericfilters.WithHTTPLogging(handler)
 	handler = genericapifilters.WithRequestInfo(handler, c.RequestInfoResolver)
 	handler = genericapifilters.WithRequestReceivedTimestamp(handler)
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServerTracing) {
+		handler = genericapifilters.WithTracing(handler, c.TracerProvider)
+	}
```

Comment:
just to double check, from the KEP:

> For incoming requests, this would go below WithRequestInfo in the filter stack, as it must be after authentication and authorization, before the panic filter, and is closest in function to the WithRequestInfo filter.

above/below are a little ambiguous... this constructs the filters "inside out"... the last filter added here runs first. Is the intent to run this filter before almost all other filters, including authn/authz and request received timestamp?

Reply:
I'm having trouble remembering the original conversation, but here is the comment I left: https://github.com/kubernetes/enhancements/pull/1458#discussion_r426961816.  The WithRequestInfo reference may not be as relevant anymore since it was recently moved in #102934.  I think the intention is that it is executed _after_ authentication, so I've placed it to be executed after (above in code) the authentication filter.